src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -16,44 +16,67 @@
 
 package io.confluent.connect.elasticsearch;
 
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 import org.apache.kafka.common.config.AbstractConfig;
 import org.apache.kafka.common.config.ConfigDef;
 import org.apache.kafka.common.config.ConfigDef.Importance;
 import org.apache.kafka.common.config.ConfigDef.Type;
 import org.apache.kafka.common.config.ConfigDef.Width;
 
 import java.util.Map;
+import org.apache.kafka.common.config.types.Password;
 
 import static io.confluent.connect.elasticsearch.DataConverter.BehaviorOnNullValues;
 import static io.confluent.connect.elasticsearch.bulk.BulkProcessor.BehaviorOnMalformedDoc;
 
 public class ElasticsearchSinkConnectorConfig extends AbstractConfig {
 
+  // Connector group
   public static final String CONNECTION_URL_CONFIG = "connection.url";
   private static final String CONNECTION_URL_DOC =
       "List of Elasticsearch HTTP connection URLs e.g. ``http://eshost1:9200,"
       + "http://eshost2:9200``.";
+  private static final String CONNECTION_URL_DISPLAY = "Connection URLs";
+
   public static final String CONNECTION_USERNAME_CONFIG = "connection.username";
   private static final String CONNECTION_USERNAME_DOC =
       "The username used to authenticate with Elasticsearch. "
       + "The default is the null, and authentication will only be performed if "
       + " both the username and password are non-null.";
+  private static final String CONNECTION_USERNAME_DISPLAY = "Connection Username";
+  private static final String CONNECTION_USERNAME_DEFAULT = null;
+
   public static final String CONNECTION_PASSWORD_CONFIG = "connection.password";
   private static final String CONNECTION_PASSWORD_DOC =
       "The password used to authenticate with Elasticsearch. "
       + "The default is the null, and authentication will only be performed if "
       + " both the username and password are non-null.";
+  private static final String CONNECTION_PASSWORD_DISPLAY = "Connection Password";
+  private static final String CONNECTION_PASSWORD_DEFAULT = null;
```

Comment:
Should this have a default? or should it be `Importance.HIGH` and a required config?